frontend/src/components/episode/helpers/get-current-time.helper.ts

```diff
@@ -0,0 +1,9 @@
+import { PlayerRef } from 'components/common/player/player';
```

Comment:
rename to `frontend/src/components/episode/helpers/get-current-time/get-current-time.helper.ts`